boardconfig/boardconfig_test.go

```diff
@@ -167,6 +168,15 @@ func TestCheckBoardConfig(t *testing.T) {
 		t.Errorf("Wrong board config version: %s", vendorVersion)
 	}
 
+	vendorVersion, err = boardConfig.GetBoardConfigVersion(json.RawMessage(validBoardConfig))
```

Comment:
Better to move to TestUpdateBoardConfig?

Reply:
done